sympy/polys/polytools.py

```diff
@@ -61,8 +61,44 @@
 
 @public
 class Poly(Expr):
-    """Generic class for representing polynomial expressions. """
+    """
+    Generic class for representing and operating on polynomial expressions.
+    Subclasses Expr class.
+
+    Examples
+    ========
+
+    >>> from sympy import Poly
+    >>> from sympy.abc import x, y
+
+    Create a univariate polynomial:
+
+    >>> Poly(x*(x**2+x-1)**2)
+    Poly(x**5 + 2*x**4 - x**3 - 2*x**2 + x, x, domain='ZZ')
+
+    Create a univariate polynomial with specified domain:
 
+    >>> Poly(x*(x**2+x-1)**2, domain='R')
+    Poly(1.0*x**5 + 2.0*x**4 - 1.0*x**3 - 2.0*x**2 + 1.0*x, x, domain='RR')
+
+    Create a multivariate polynomial:
+
+    >>> Poly(y*x**2+x*y+1)
+    Poly(x**2*y + x*y + 1, x, y, domain='ZZ')
+
+    Create a univariate polynomial, where y is a constant:
+    Note: You can then evaluate this polynomial as a function of y.
+
+    >>> Poly(y*x**2+x*y+1,x)
+    Poly(y*x**2 + y*x + 1, x, domain='ZZ[y]')
+
+    See Also
+    ========
+
```

Comment:
you can remove the blank line here. Most of the time you have to have it there, but with "See Also" you don't.